dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/clients/common/OidcAuthorizationRequest.java

```diff
@@ -226,9 +226,18 @@ void addOptionalParameters(AuthorizationRequestParameters authzParameters, OidcC
             addPkceParameters(pkceCodeChallengeMethod, state, authzParameters);
         }
 
+        // LIBERTY-I-61: START
```

Comment:
We don't want to have comments like this; references to things like Aha ideas or even GitHub issues don't need to be in the code itself.